doc/shared/sunadaptcontroller/SUNAdaptController_MRIHTol.rst

```diff
@@ -0,0 +1,165 @@
+..
+   Programmer(s): Daniel R. Reynolds @ SMU
+   ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2024, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _SUNAdaptController.MRIHTol:
+
+The SUNAdaptController_MRIHTol Module
+=======================================
+
+.. versionadded:: x.y.z
+
+Mathematical motivation
+-----------------------
+
+The MRIHTol implementation of the SUNAdaptController class,
+SUNAdaptController_MRIHTol, implements a general structure for telescopic
+multirate temporal control.  A SUNAdaptController_MRIHTol object is constructed
+using two single-rate controller objects, *HControl* and *TolControl*.  The
+MRIHTol controller assumes that overall solution error at a given time scale
+results from two types of error:
+
+#. "Slow" temporal errors introduced at the current time scale,
+
+   .. math::
+      \varepsilon^s_{n} = C(t_n) \left(h_n^s\right)^P,
+      :label: slow_error_assumption
+
+   where :math:`C(t)` is independent of the current time scale step size :math:`h^s`
+   but may vary in time.
+
+#. "Fast" errors introduced through calls to the next-fastest ("inner") solver,
+   :math:`\varepsilon^f_{n}`.  If this inner solver is called to evolve IVPs over
+   time intervals :math:`[t_{0,i}, t_{F,i}]` with a relative tolerance
+   :math:`\text{reltol}_n^f`, then it will result in accumulated errors over these
+   intervals of the form
+
+   .. math::
+      \varepsilon^f_{n} = c(t_n) \left(\text{reltol}_n^f\right) \left(t_{F,i}-t_{0,i}\right),
+
+   where :math:`c(t)` is independent of the tolerance or subinterval width but may vary in
+   time, or equivalently,
+
+   .. math::
+      \varepsilon^f_{n} = \kappa(t_n) \left(\text{tolfac}_n^f\right),
+      :label: inner_solver_assumption
+
+   where :math:`\text{reltol}_n^f = \text{reltol}^s \text{tolfac}_n^f`,
+   :math:`\text{reltol}^s` is the relative tolerance that was supplied to the
+   current time scale solver, and where
+   :math:`\kappa(t_n) = c(t_n) \text{reltol}^s \left(t_{F,i}-t_{0,i}\right)` is
+   independent of the relative tolerance factor, :math:`\text{tolfac}_n^f`.
+
+Single-rate controllers are constructed to adapt a single order parameter, e.g.,
```

Comment:
I found this a bit confusing. It sounds like controllers are adapting order rather than step size.